src/arch/dotproductavx.cpp

```diff
@@ -14,6 +14,9 @@
 // WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 // See the License for the specific language governing permissions and
 // limitations under the License.
+//
+// Copyright (c) 2017, RRZE-HPC Erlangen
```

Comment:
?

Reply:
That's not my license. Did you read [my comment](https://github.com/tesseract-ocr/tesseract/pull/2106/commits/aada1861f4d70d5815e748ed9ff926f495440f4c)?

Reply:
Did you read [my comment](https://github.com/tesseract-ocr/tesseract/pull/2106/commits/aada1861f4d70d5815e748ed9ff926f495440f4c)?

Reply:
This pull request was sent as a request for comments (RFC) and is not ready to get merged. If we don't get a compatible license from RRZE-HPC Erlangen, we can rewrite that small part of the code.

Reply:
Ping @RRZE-HPC because of the license issue. Could we reuse your code under Apache-2.0, too?

Reply:
try pinging individuals...

Reply:
That's only a detail (AVX dot product with float) of this pull request. I can even talk to the people in Erlangen.

For SSE we will have to provide own code (maybe based on the assembler code generated by a highly optimizing compiler),  and if needed we could replace the AVX code, too.

More important would be feedback on the performance on various platforms and whether using float instead of double changes the OCR results (it did not in our tests).

Noah is currently working on using float for the training part, too, so hopefully the training process can be accelerated a lot in the future.